reference/02-Themes/navigation.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_label: Navigation
+sidebar_position: 5
+---
+
+# Navigation in Block-based Themes
+
+Block-based themes changed how navigation areas are built. Instead of using `wp_nav_menu()` in your theme, you now use blocks to create navigation areas. Each navigation menu now has a post in a hidden core post type called `wp_navigation`. These posts then contain the block markup for the navigation menu.
+
+## Working with Navigation Menus
+
+The core navigation block stores the ID of the navigation post in its attributes. This way the block knows which navigation menu to display.
+
+Sadly, because of this reliance on the ID we loose the concept of navigation areas. In classic themes you could register a navigation area and then assign a menu to it in the Customizer. This is not possible anymore with block-based themes.
```

Comment:
```suggestion
Sadly, because of this reliance on the ID we lose the concept of navigation areas. In classic themes you could register a navigation area and then assign a menu to it in the Customizer. This is not possible anymore with block-based themes.
```